tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/ChunkMetadata.java

```diff
@@ -171,12 +174,22 @@ public void setVersion(long version) {
     this.version = version;
   }
 
-  public long getDeletedAt() {
-    return deletedAt;
+  public List<Pair<Long, Long>> getDeleteRangeList() {
+    if (deleteRangeList == null) {
```

Comment:
em... if so, you still generate many ArrayList instances ...
If you do not want to check whether the list is null, use `Collections.emptyList` instead

Reply:
Fixed